lib/DirectAdmin_API.php

```diff
@@ -47,14 +47,32 @@ public function __construct($host = null, $login = null, $pass = null, $secure =
 			$this->_setupLocalConnection();
 	}
 	
-	public function getDomainsMxRecords($domain){
+	public function getDomainsMxRecords($domain, $username){
```

Comment:
__[phpmd]__
* The method getDomainsMxRecords() has a Cyclomatic Complexity of 10. The configured cyclomatic complexity threshold is 10.

<!-- Comment by [SideCI](https://sideci.com) -->